scripts/github-status/github-status.rb

```diff
@@ -116,7 +116,8 @@ def get_pull_requests(state, status = [])
       pulls.partition do |p|
         next true unless @repository == "SUSE-Cloud/automation"
         pf = @client.pull_request_files(@repository, p[:number]).select do |f|
-          f[:filename] =~ %r{scripts/(mkcloud|qa_crowbarsetup\.sh|lib/.*)$}
+          f[:filename] =~ %r{scripts/(mkcloud|qa_crowbarsetup\.sh|lib/.*)$} ||
+          f[:filename] =~ %r{scripts/jenkins/log-parser/}
```

Comment:
> Yes, that was the intention.

But why do you introduce the new job then?

IMHO we should either have github-status trigger the new job automatically or exclude the default mkcloud run from PRs to this file to be able to build this new job manually.


> That sounds great - we could use it to run the openstack-mkcloud-log-parser-test job which this PR introduces. 

I thought that that was what my initial proposal said :)

> Notice how I parametrised [...]

You are a Jenkins master.